tests/test_pkce.py

```diff
@@ -12,16 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from unittest import TestCase
+import unittest
```

Comment:
Use `unittest2` for 2.6 support.

Won't be needed for long, but it is for now.